Source/Document Structure/SvgScript.cs

```diff
@@ -11,46 +9,43 @@ namespace Svg
     [SvgElement("script")]
     public class SvgScript : SvgElement
     {
-
-        public string Script 
+        public string Script
         {
             get { return this.Content; }
             set { this.Content = value; }
         }
 
-
         [SvgAttribute("type")]
-        public string ScriptType 
-        { 
-            get { return GetAttribute<string>("type", false); } 
-            set { Attributes["type"] = value; } 
+        public string ScriptType
+        {
+            get { return GetAttribute<string>("type", false); }
+            set { Attributes["type"] = value; }
         }
 
-
         [SvgAttribute("crossorigin")]
-        public string CrossOrigin 
+        public string CrossOrigin
         {
-            get { return GetAttribute<string>("crossorigin",false); }
+            get { return GetAttribute<string>("crossorigin", false); }
             set { Attributes["crossorigin"] = value; }
         }
 
-        [SvgAttribute("href")]
+        [SvgAttribute("href", SvgAttributeAttribute.XLinkNamespace)]
```

Comment:
FYI: Deprecated in SVG2.

https://www.w3.org/TR/SVG2/linking.html#XLinkRefAttrs